### PR TITLE
Fix container init (due to AiiDA removal)

### PR DESCRIPTION
**Pull request by knopki**
Container initialization is broken because of schema file rename in [#5db5de7](https://github.com/basf/metis-backend/commit/5db5de7ff779b28f686e7146e5d2fc1d4aa61fba)
+
Container initialization is broken because of aiida removal in [#bf11bfb](https://github.com/basf/metis-backend/commit/bf11bfbc0e2a48aebe2bf9a9b21d6573aa2f153d)

### CODE REVIEW

**Comment by blokhin** [comment]
@knopki AiiDA was just temporarily removed due to very cumbersome installation, generally, we’re still relying on it in the long term

File: .docker/init-aiida.sh
Comment (blokhin) [inline .docker/init-aiida.sh:1]: @knopki I think we will need it later

File: .docker/s6-rc.d/wait-rmq/up
Comment (blokhin) [inline .docker/s6-rc.d/wait-rmq/up:1]: @knopki I think we will need all these beasts later

**Comment by blokhin** [comment]
Currently the best option would be not to account for AiiDA but still check if it’s available (and be able to install it, if the user requires its workflows).

**Comment by knopki** [comment]
> @knopki AiiDA was just temporarily removed due to very cumbersome installation, generally, we’re still relying on it in the long term

When AiiDA returns, we will return the scripts to install it. Right now the installation is broken anyway (although it can be fixed).

**Comment by knopki** [comment]
> Currently the best option would be not to account for AiiDA but still check if it’s available (and be able to install it, if the user requires its workflows).

I don't think it's any better. It adds to the complexity.

**Review by blokhin — APPROVED** [review]

**Pull request merged**